src/backlinks/handler.js

```diff
@@ -16,6 +16,22 @@ import {
 import { composeAuditURL } from '@adobe/spacecat-shared-utils';
 import AhrefsAPIClient from '../support/ahrefs-client.js';
 import { retrieveSiteBySiteId } from '../utils/data-access.js';
+import { fetch } from '../support/utils.js';
+
+async function filterOutValidBacklinks(backlinks, log) {
+  const isStillBrokenBacklink = async (backlink) => {
+    try {
+      const response = await fetch(backlink.url_to);
+      return !response.ok;
```

Comment:
it might be a good idea to `log.warn` 429s (maybe all 4XXs except 404s) for those sites behind CDN protections because of sending multiple HTTP requests from `Promise.all`.

Reply:
this is a good idea as we then can configure graphs and alerts on coralogix

Reply:
I incorporated the `log.warn`.